modules/tests/src/smithy4s/tests/PizzaSpec.scala

```diff
@@ -312,27 +312,59 @@ abstract class PizzaSpec
     )
   }
 
-  routerTest("HEAD request should have empty body") { (client, uri, log) =>
+  routerTest(
+    "HEAD request should have empty body - content length from just brackets"
+  ) { (client, uri, log) =>
     for {
       res <- client.send[String](
         HEAD((uri / "head-request")),
         log
       )
     } yield {
       val (code, headers, body) = res
+      val expectedHeaders = Map(
+        "Test" -> List("test"),
+        // response would have been `{}` if this was a GET
+        "Content-Length" -> List("2"),
+        "Content-Type" -> List("application/json")
+      )
+      val containsProperHeaders =
+        expectedHeaders.forall(h => headers.get(h._1).isDefined)
```

Comment:
should you also use `expectedHeaders.forall(h => headers.get(h._1).contains(h._2))` here?

Reply:
Ah yeah, thanks will update that now